### PR TITLE
Add a script for testing deployment on mainnet

### PR DESCRIPTION
**Pull request by theref**

### CODE REVIEW

**Comment by manumonti** [comment]
Some of the commits of this PR has been already merged in #13, so it's hard to review since there is a lot of noise.

Could you please rebase theref:deploy branch to threshold-network:main in order to this PR only shows the actual changes?

File: deploy/deploy.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ module.exports = async ({ deployments, getNamedAccounts }) => {
     console.log('network id ', await getChainId());
 
     const {deploy} = deployments;
-    const {deployer, tokenContract, rewardsHolder} = await getNamedAccounts();
-    const args = [tokenContract, rewardsHolder];
+    const {deployer, tokenContract, rewardsHolder, owner} = await getNamedAccounts();
+    const args = [tokenContract, rewardsHolder, owner];
```
Comment (manumonti) [inline deploy/deploy.js:10]: If we merge #18, the arguments of the constructor must include TokenStaking contract address.

**Review by manumonti — CHANGES_REQUESTED** [review]
As I see, we should merge finish and merge #18 before merging this PR.

File: deploy/deploy.js (hunk 1 above)
Comment (manumonti) [reply deploy/deploy.js:10]: Finally #18 won't be merged

https://github.com/threshold-network/merkle-distribution/pull/18#issuecomment-1184720318

**Review by manumonti — APPROVED** [review]
LGTM

**Review by cygnusv — APPROVED** [review]

**Pull request merged**